fixtures/Client.yaml

```diff
@@ -0,0 +1,31 @@
+App\Entity\Client:
+    client1:
+        companyName: Citron
+        address: <address()>
+        siretNumber: <fr_FR:siret()>
+        phoneNumber: <fr_FR:phoneNumber()>
```

Comment:
No template?

Reply:
This comment holds for every fixture file ^^

Reply:
Done ! :)